docs/spec/src/protocol/architecture/write-and-read-workflow.md

```diff
@@ -0,0 +1,57 @@
+## Write and Read Workflow
+
+This page provides an overview of the workflow for writing data to and reading data from EigenDA. The workflow is illustrated in the diagram below.
+
+![image](../../assets/write-and-read-workflow.png)
+
+**Notes:**
+* The "end user" for writing and the "end user" for reading can be the same entity. They are shown separately in the diagram for clarity.
+* We are planning to build full nodes that will perform the disperser's functionality plus additional duties.
+
+
+### Write
+
+When a user writes data to EigenDA (in the form of a blob), the blob is encoded into chunks and distributed to the validators in accordance with the [Chunk Assignment Logic](./assignment.md). After enough validators have acknowledged receipt of their chunks and returned their signatures to the disperser, the disperser aggregates the signatures into a data availability (DA) certificate and sends it to the user upon request.
+
+The write process follows the sequence below. The labels in parentheses (e.g., W1, W2) correspond to the steps shown in the diagram above.
+
+1. **Disperser Receives Blob (W1, W2, W3).**
+   The disperser receives a blob consisting of a `BlobHeader` and `BlobData`. As a precaution, the disperser can validate the `PaymentMetadata` contained in the `BlobHeader` to ensure that the blob is properly funded, and that the KZG commitments in the `BlobHeader` are correct.
+
+2. **Disperser Encodes Blob (W6, W7).**
+   The disperser references the Chunk Assignment Logic to translate the `BlobHeader` into a set of `EncodingParams`. The disperser then encodes the blob according to the [Encoding Module](./encoding.md) and the `EncodingParams` to produce a collection of encoded `Chunk`s.
+
+3. **Disperser Serves Chunks.**
+   The disperser makes the encoded chunks available via the relay's `GetChunks` interface. This is an authenticated and rate-limited interface where each validator can only request its allocated amount of data.
```

Comment:
> can only request its allocated amount of data.

Technically not true today, but it will eventually be true. IMO, ok to leave sentence it the way it is.